### PR TITLE
Changing to code block of type shell to allow copying of code in Github

### PR DESCRIPTION
**Pull request by tysonwolker**
Changes proposed in this pull request:

- Changing to code block of type `sh` to allow copying of code in Github
### Screenshots
image
image

### CODE REVIEW

File: .github/README.md [hunk 1]
```diff
@@ -6,15 +6,19 @@ A good description is clear, short, and to the point. Describe the purpose of th
 
 Describe how to install the project locally then **DELETE THIS LINE**. For example:
 
-1. `git clone git@github.com:qctrl/[REPOSITORY].git`
-1. `cd [REPOSITORY]`
-1. `install...`
+```cmd
```
Comment (robertlove) [inline .github/README.md:9]: @tysonwolker I like where this is going. I'd like to know more about the decision to use `cmd`.

- Look at [GitHub Syntax Highlighting](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting).
- This links to [the languages YAML file](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml).
- Note that `cmd` denotes a [Batchfile](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L503).
- Note that `sh` denotes a [Shell](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L6000).
- See also this [Markdown code styleguide](https://arcticicestudio.github.io/styleguide-markdown/rules/code.html#escape-newlines-in-terminal-commands) - which I'm not saying we should adopt - but it's a point of reference.

So where did the choice of `cmd` come from?

**Review by robertlove — CHANGES_REQUESTED** [review]

**Comment by robertlove** [comment]
> Adding cmd to shell code blocks adds the ability to copy code from markdown files but additionally takes up full width of a Markdown file.

@tysonwolker it's not the use of `cmd` that renders the code full width - it's the use of triple backticks. In Markdown, single backticks produce inline code (i.e. `<code>`) and triple backticks produce block code (i.e. `<pre>`).

See here https://commonmark.org/help/

**Comment by tysonwolker** [comment]
> > Adding cmd to shell code blocks adds the ability to copy code from markdown files but additionally takes up full width of a Markdown file.
> 
> @tysonwolker it's not the use of `cmd` that renders the code full width - it's the use of triple backticks. In Markdown, single backticks produce inline code (i.e. `<code>`) and triple backticks produce block code (i.e. `\<pre\>`).
> 
> See here [commonmark.org/help](https://commonmark.org/help/)

Yes thats right its the triple use of ` that makes it full width.

File: .github/README.md (hunk 1 above)
Comment (tysonwolker) [reply .github/README.md]: @robertlove thanks [this](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml) was a good share. Agreed `sh` is definitely the right type.

Comment (robertlove) [reply .github/README.md]: @tysonwolker super! WDYT about using the full version `shell` rather than the alias `sh`?

**Review by robertlove — APPROVED** [review]

File: .github/README.md (hunk 1 above)
Comment (cfarrend) [reply .github/README.md]: The Markdown code styleguide looks like a great resource.

**Pull request merged**